### PR TITLE
Update Changes.app

### PR DESCRIPTION
**Pull request by texastoland**

### CODE REVIEW

File: Casks/changes.rb [hunk 1]
```diff
@@ -2,11 +2,18 @@
   version :latest
   sha256 :no_check
 
-  url 'http://bitbq.com/changes/download.php'
+  url 'https://bitbq_changes.s3.amazonaws.com/changes.zip'
```
Comment (vitorgalvao) [inline Casks/changes.rb]: Needs a `# amazonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5).

Comment (dnalot) [inline Casks/changes.rb]: Fixed. Is it required for every new cask in this tap?

Comment (vitorgalvao) [inline Casks/changes.rb]: It’s required when `url` is from a different host than `homepage`. Is to guarantee we’re aware of that discrepancy, but it is as the dev intended.

**Comment by vitorgalvao** [comment]
Looks good, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by dnalot** [comment]
Done. I hadn't just in case you pulled it.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**